Assets/Scripts/SS3D/Data/Assets.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System.Collections.Generic;
+using Coimbra;
+using SS3D.Data.AssetDatabases;
+using SS3D.Data.Enums;
+using SS3D.Logging;
+using UnityEngine;
+using Object = UnityEngine.Object;
+
+namespace SS3D.Data
+{
+    /// <summary>
+    /// Assets is used to load assets using enums or ints, with the addition of an enum generator for easy "id" of items.
+    /// </summary>
+    public static class Assets
+    {
+        /// <summary>
+        /// All loaded databases.
+        /// </summary>
+        private static readonly Dictionary<string, AssetDatabase> Databases = new();
+
+        // IMPORTANT: All database getters have to be added manually. For now.
+
+        public static Sprite Get(InteractionIcons icon)
+        {
+            return GetDatabase(nameof(InteractionIcons)).Get<Sprite>((int)icon);
+        }
+
+        public static GameObject Get(ItemIds itemIdId)
```

Comment:
Isn't there a better name for itemIdId ? It's not clear what it is from the name